internal/api/urls.go

```diff
@@ -32,6 +33,10 @@ func isLocalhost(host string) bool {
 	return addr.IsLoopback()
 }
 
+func isUsingStella (host string) bool {
```

Comment:
I would have preferred having a `map[string]bool` with key = hostname for whitelisting, instead one func per host.